src/store/modules/swap/ethBancor.ts

```diff
@@ -563,6 +563,64 @@ const determineConverterType = (
   throw new Error("Failed to determine the converter type");
 };
 
+const getHistoricFees = async (
+  id: string,
+  converterAddress: string,
+  network: EthNetworks
+): Promise<PreviousPoolFee[]> => {
+  const w3 = getWeb3(network);
+  const contract = buildV28ConverterContract(converterAddress, w3);
+  const { blockHoursAgo, currentBlock } = await blockNumberHoursAgo(
+    24,
+    network
+  );
+  const history = [];
+  const chunkSize = 2000;
+
+  let startBlock = blockHoursAgo;
+  let endblock = startBlock + chunkSize;
+
+  while (startBlock < currentBlock) {
+    const options = {
+      fromBlock: startBlock,
+      toBlock: endblock
+    };
+
+    const events = await contract.getPastEvents("ConversionFeeUpdate", options);
+
+    history.push(
+      ...events.map(e => ({
+        id,
+        oldDecFee: e.returnValues["_prevFee"] / (10000 * 100),
```

Comment:
I've previously defined a ```decToPpm``` function, define a ```ppmToDec``` as well. ```(10000 * 100)``` won't be immediately obvious to a new reader as to what this is.

Reply:
done